boards/common/remote/Makefile.include

```diff
@@ -24,6 +24,12 @@ FLASHFILE ?= $(BINFILE)
 DEBUGGER_FLAGS = $(BINDIR) $(ELFFILE)
 OBJDUMPFLAGS += --disassemble --source --disassembler-options=force-thumb
 
+# remote use cc2538-bsl attaching to the serial PORT to reset the board. If
```

Comment:
This comment is hard to read. Maybe (with typos fixed):
`remote uses cc2538-bsl attached to the serial PORT to reset the board. If 'make reset' is called when the terminal is opened, the device switches to bootloader mode and the next commands will fail, leaving the device in bootloader mode. To avoid this problem, the use of reset as sync method is disabled.`